LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -128,6 +128,7 @@
     "web-platform-tests/css/css-grid/parsing/": "import",
     "web-platform-tests/css/css-highlight-api": "import",
     "web-platform-tests/css/css-images": "import",
+    "web-platform-tests/css/css-images/image-set/": "import",
```

Comment:
Please undo this. It’s redundant with the previous line

Reply:
I just undo the line though, I'm wondering when the import expectations should be added, I noticed some other places might also have child folders in the import-expectations.json like following

https://github.com/WebKit/WebKit/blob/b50c5911015252922d4511ec940e88b1dba20354/LayoutTests/imported/w3c/resources/import-expectations.json#L155-L158

Reply:
yeah, it's an unintended side effect of the script, ideally we should clean up the others as well